tox.ini

```diff
@@ -6,7 +6,7 @@ recreate = True
 passenv = DISPLAY MOZ_HEADLESS TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
 deps = pipenv
 commands =
-    pipenv install --dev --skip-lock
+    pipenv install -r pipenv.txt --dev --skip-lock
```

Comment:
pipenv is installed on line 7, and this line is running pipenv. Here, you're asking pipenv to install itself. Can you modify line 7 to refer to the new pipenv.txt file?

Reply:
Done